app/models/event/question.rb

```diff
@@ -88,7 +88,7 @@ def label
   end
 
   def global?
-    event_id.blank?
+    event.blank?
```

Comment:
Das verstehe ich nicht. Ich dachte immer, dass relation_id prüfen besser ist, weil es nicht noch das Modell lädt. Ist was aufgrund STI im Event nötig?

Reply:
Grundsätzlich ja, aber in diesem Fall wird die Question zusammen mit dem Event erstellt. event_id ist dementsprechend leer wenn validiert wird, aber die Referenz auf das Event-Objekt ist schon da.

Reply:
Oh, das ein wichtiger Grund, es so zu lösen. Ich erkenne es leider nicht ganz: ist das durch die Spec-Änderung im Folgecommit mit abgedeckt? Ich kann mir gut vorstellen, dass das sonst gerne mal wieder kaputt-optimiert wird. Wenn ja, würde ich es beim Merge squashen, damit es in der Historie als eine Änderung sichtbar ist.